src/Google.Cloud.Functions.Framework.Tests/GcfEvents/EventDeserializationTest.cs

```diff
@@ -144,45 +140,38 @@ public async Task LegacyEvents()
         [InlineData("firebase-dbdelete1.json")]
         [InlineData("firebase-dbdelete2.json")]
         public Task FirebaseDatabaseEvent_ParseWithoutError(string resource) =>
-            ConvertAndDeserialize<DocumentEventData>(resource);
+            ConvertAndDeserialize<ReferenceEventData>(resource);
 
         [Fact]
         public async Task FirebaseDatabaseEvent_NullData()
         {
-            var firebaseEvent = await ConvertAndDeserialize<DocumentEventData>("firebase-db1.json");
-            Assert.Null(firebaseEvent.Data);
-        }
-
-        [Fact]
-        public async Task FirebaseDatabaseEvent_Wildcards()
```

Comment:
I suppose wildcards are not special anymore, right?

Reply:
They're not part of the event payload. (They were in an odd place in the GCF event, and we've decided not to include them in the CloudEvent representation.)